Podfile

```diff
@@ -16,9 +16,9 @@ use_frameworks!
 # - `{ :specHash => {sdk spec hash}` to depend on specific pod options (:git => …, :podspec => …) for MatrixSDK repo. Used by Fastfile during CI
 #
 # Warning: our internal tooling depends on the name of this variable name, so be sure not to change it
-# $matrixSDKVersion = '= 0.24.0'
+$matrixSDKVersion = '= 0.24.0'
 # $matrixSDKVersion = :local
-$matrixSDKVersion = { :branch => 'develop'}
+# $matrixSDKVersion = { :branch => 'develop'}
 # $matrixSDKVersion = { :specHash => { git: 'https://git.io/fork123', branch: 'fix' } }
```

Comment:
@langleyd what about the podfile and the checksum? I see they were pushed with the develop branch for the matrix sdk as default, should I keep it that way or restore it to the original state as I did here?